ipy2to3_other.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Converts an IPython notebook written in python2 to python3.
+
+import io
+import json
+import os
+import subprocess
+import tempfile
+
+def is_python_code_cell(cell):
+    return cell['cell_type'] == "code"# and cell['language'] == "python"
+
+def is_magic(line):
+    line = line.strip()
+    if len(line) == 0: return False
+    return line[0] in ['%', '!', '?'] or line[-1] == '?'
+
+# Empties magic lines and returns where they were and what they are
+def replace_magic_lines(lines):
+    magic_lines = []
+    for i, line in enumerate(lines):
+        if is_magic(line):
+            magic_lines.append((i, lines[i]))
+            lines[i] = "\n"
+    return magic_lines
+
+def convert_ipy2to3(json_to_convert):
+    #for worksheet in json_to_convert['worksheets']:
+    worksheet=json_to_convert
+    if True:
+        code_cells = filter(is_python_code_cell, worksheet['cells'])
+        for cell in code_cells:
+            magic = replace_magic_lines(cell['source'])
+            file_name = None
+            with tempfile.NamedTemporaryFile(
+                    mode = "w", delete = False) as ostream:
+                ostream.writelines(cell['source'])
+                file_name = ostream.name
+            cmd2to3 = ["C:/Anaconda3_64/Scripts/2to3.exe", "--nobackups", "--write", file_name]
```

Comment:
If that's the case then I'd suggest using the original '2to3' on its own which is less confusing and less specific to windows users